### ISSUE TITLE
Low-level interface error with SRK and PSmass_INPUTS

### ISSUE THREAD

**Issue by colinheye** [CoolProp/CoolProp#1966]
I receive a ValueError when attemping to update the low-level interface for parahydrogen with PSmass_INPUTS

CoolProp.AbstractState("SRK","parahyd").update(PSmass_INPUTS, P1, S1)

The exact error is "ValueError: type not set"

Is this functionality not available or is this an issue with P1 and S1 being out of bounds?

CoolProp python wrapper version 6.3.1dev on Windows

**Comment by ibell** [comment]
The error message could be better, but I believe the short answer is that
PS inputs are not supported for SRK?  You'd be welcome to contribute to add
that functionality.

**Comment by colinheye** [comment]
There may be interest in contributing. Any quick tips on what changes need to be made?

**Comment by ibell** [comment]
Basically the algorithms in the HEOS backend need to be adapted to also
support the cubic backends.  Since everything is class-based, most of the
code needs to be just modified, and hopefully only rather minor changes
would be needed; that was the point of developing the structure in this way
when we developed version 5.

**Comment by colinheye** [comment]
Thanks. Would you anticipate RKS inverse function calls being computationally faster than HEOS?

**Comment by ibell** [comment]
Yes, I would expect so, the calls to the EOS with SRK are faster than with
the multiparameter EOS.
